client/dive-common/store/settings.ts

```diff
@@ -75,8 +87,8 @@ export default function setup(allTypes: Ref<Readonly<string[]>>) {
       clientSettings.trackSettings.newTrackSettings.type = 'unknown';
     }
   });
-  watch(clientSettings, saveSettings);
 }
+watch(clientSettings, saveSettings);
```

Comment:
If not the only time clientSettings get watched are inside of the Viewer.vue, when it loads in all the types.  This is needed so that a user changing the number in the databrowser has their setting persist between views.

Without this if you were to go to the data browser and set it to 50 (nothing gets saved) and then go to the shared Tab it would reset back to the default.  That watcher before was tied into the allTypes update because we only had settings that persisted inside of the annotator, not throughout the data browser.

Reply:
OK, now I understand, thank you.